bucket/mpv.net.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://github.com/stax76/mpv.net/",
+    "description": "A modern media player for Windows that works just like mpv",
+    "license": "MIT",
+    "version": "4.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/stax76/mpv.net/releases/download/4.0/mpv.net-portable-x64-4.0.0.0.7z",
```

Comment:
Shouldn’t the version be 4.0.0.0, not just 4.0?

Reply:
As suggested by Ash258, the main version number is enough.😄️